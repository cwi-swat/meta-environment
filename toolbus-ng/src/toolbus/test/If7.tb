process OpenModule is
let Path : str
in
    Path := "abc" .
    if false then
      printf("wrong\n")
    else
      let
        Term: path
      in
         Term := quote(path(Path)).
         printf("Setting term: %t\n", Term)
      endlet
    fi
  . printf("Done\n")
endlet

toolbus(OpenModule)
