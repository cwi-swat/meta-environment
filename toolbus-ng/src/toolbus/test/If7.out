Setting term: path("abc")
Done
